umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/echo10.clj

```diff
@@ -187,10 +187,10 @@
                  ;; expectes a string in %9.2f format, so we have to
                  ;;ignore 'Free', 'Gratis', etc.
                  (catch NumberFormatException e)))]
-     [:DataFormat (-> c
-                      :ArchiveAndDistributionInformation
-                      :FileDistributionInformation
-                      first :Format)]
+     ; Go through all of the FileDistributionInformation elements and set the
```

Comment:
for comments on one line please use ;; instead.  Please check out https://wiki.earthdata.nasa.gov/display/CMR/Coding+Style+Guidelines when you can.

Reply:
fixed.  Thanks.

Reply:
fixed.